### PR TITLE
LPS-133441

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**
https://issues.liferay.com/browse/LPS-133441

### CODE REVIEW

**Comment by ozysouza** [comment]
This pull request refer to https://issues.liferay.com/browse/LPS-133441

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 1]
```diff
@@ -1296,6 +1296,87 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-133280. Poshi Automation for Bug LPS-129830 Under certain situations, unable to remove multiple manually set assets."
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:1299]: Change for "This is a test for LPS-129830. Under certain situations, unable to remove multiple manually set assets."

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 2]
```diff
@@ -1296,6 +1296,87 @@ definition {
...
+	@priority = "4"
+	test ViewRemovedWebContentInAssetPublisherConfiguration {
+		task ("Go to GogoShell in System") {
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:1302]: Maybe we can group these four steps in one, like: "Stop and start module controller portlet from GogoShell"

**Review by david-gutierrez-mesa — COMMENTED** [review]
Good job! Just some small comments

**Comment by ozysouza** [comment]
New PR: https://github.com/david-gutierrez-mesa/liferay-portal/pull/160

**Comment by ozysouza** [comment]
ci:close

**Comment by david-gutierrez-mesa** [comment]
ci:close

**Pull request closed without merging**